software/main.py

```diff
@@ -400,12 +572,17 @@ class State(Enum):
 
                 #print(str(xboxcont.joystick_left_y) + " / " + str(xboxcont.joystick_left_x))
 
-                speedy = joyY * 5
-                speedx = joyX * 5
-                speedr = joyRightX * 20
+                speedy = joyY * 13
+                speedx = joyX * 13
+                speedr = joyRightX * 30
                 speedthrow = joyRTrig * 4500
```

Comment:
Change in the variables naming format - previously you used snake_case. Now it is lowercase. Please unify